### DISCUSSION TITLE
Use accent color for current window selection

### DISCUSSION THREAD

**Discussion by marcinjahn** [paperwm/PaperWM#907] [Ideas]
As accent colors are coming up soon in Gnome 47, I think it's worth to think about adding an option to set accent color as the default "current window" border color. I guess it'd make a lot of sense to go with that.

**Comment by jtaala** [comment]
I think this is a good idea.  Users can still override colour via `~/.config/paperwm/user.css` if they want to.

**Comment by jtaala** [comment]
We shouldn't need to do anything here (PaperWM will already use the accent colour by default since we inherit `tile-preview` styling, which in Gnome 47 will use the accent colour):

https://gitlab.gnome.org/GNOME/gnome-shell/-/blob/main/data/theme/gnome-shell-sass/widgets/_misc.scss#L50

**Reply by marcinjahn** [reply]
Cool, then at least now it can be considered a conscious decision I guess 🙂
